wgsl/index.bs

```diff
@@ -90,7 +90,90 @@ that run on the GPU.
  * Features and semantics are exactly the ones of SPIR-V
  * Each item in this spec *must* provide the mapping to SPIR-V for the construct
 
-## Technical Overview TODO ## {#technical-overview}
+## Technical Overview ## {#technical-overview}
+
+WebGPU issues a unit of work to the GPU in the form of a <dfn noexport>provoking command</dfn>.
```

Comment:
My bikeshed-fu is weaker than it could be.  Teach me.  :-)
I'm also open to feedback about which definitions in this spec should be exported.   We can revisit en mass when more cross-spec work is done.

I changed this in the parent PR #1582 to be a cross-reference to "GPU command" in the WebGPU spec.